int/api/massa/version.go

```diff
@@ -0,0 +1,12 @@
+package massa
```

Comment:
why you put this in the massa package ?
folders in `int/api` correspons tout subroute
by instance int  `int/api/massa` you have the handler for the endpoint `http://....../massa/addresses`
you can put your file directly in `int/api` with its own package "version"
you can also rename `GetMassaStationVersionFunc` by `Handle` to keep keeping some coherence in the project

Reply:
thanks for the insight I made the changes